src/CodeGen.fs

```diff
@@ -1150,8 +1166,11 @@ let inline sampleFSharpClientMember serializer query queryName hasVariables useT
 let sampleFableGraphqlClient projectName clientName errorType members =
```

Comment:
Fable client implementation is broken in this PR, there should be no usage of any of the following
 - System.Net.Http
 - Microsoft.FSharp.Control
 - System.Net.Http.Json
 - System.Text.Json.Serialization
 - HttpClient
 - JsonFSharpConverter

Please revert _all_ changes made to the Fable clients since it doesn't use System.Text.Json